File: 2014-10-CSharp-Fundamentals-1/2. Primitive Data Types and Variables/StringLiterals/StringLiterals.cs
using System;

class StringLiterals
{
	static void Main()
	{
		// Here is a string literal using escape sequences
		string quotation = "\"Hello, Jude\", he said.";
		string path = "C:\\WINNT\\Darts\\Darts.exe";
		Console.WriteLine(quotation);
		Console.WriteLine(path);
		
		// Here is an example of the usage of @
		quotation = @"""Hello, Jimmy!"", she answered.";
		path = @"C:\WINNT\Darts\Darts.exe";
		Console.WriteLine(quotation);
		Console.WriteLine(path);

		Console.WriteLine(@"
			This is a multi-line text,
			represented as verbatim string.
			Note that \n and \t are printed
			as normal characters, not as
			escaping sequences.
		");
	}
}

File: 2015-10-Java-Fundamentals-1/02. Data-types-and-variables/demos/src/StringLiterals.java
import java.text.MessageFormat;

public class StringLiterals {
    public static void main(String[] args) {
// Here is a string literal using escape sequences
        String quotation = "\"Hello, Jude\", he said.";
        String path = "C:\\WINNT\\Darts\\Darts.exe";
        WriteLine(quotation);
        WriteLine(path);

        WriteLine("This is a multi-line text,\n" +
            "represented as string.");
    }

    public static void WriteLine(Object obj) {
        System.out.println(obj);
    }

    public static void WriteLine(String format, Object... obj) {
        MessageFormat form = new MessageFormat(format);
        System.out.println(form.format(obj));
    }
}
